SwiftCharts/ChartCoordsSpace.swift

```diff
@@ -118,7 +174,18 @@ public class ChartCoordsSpace {
                 layer.rect.width + chartSettings.spacingBetweenAxesY
             }, generator: generator)
     }
-    
+
+    /**
+     Uses a generator to make axis layers from axis models. This method is used for all axes.
+
+     - parameter axisModels:    The models used to generate the axis layers
+     - parameter offset:        The offset in points for the generated layers
+     - parameter pointsCreator: TODO: not sure exactly what this effectively does
```

Comment:
Maps the variable dimension (for y-axes this is x and for x-axes it's y) in multi-axes space to corresponding start and end point of axis.